src/common/util.cc

```diff
@@ -274,7 +274,7 @@ string cleanbin(bufferlist &bl, bool &base64)
 {
   bufferlist::iterator it;
   for (it = bl.begin(); it != bl.end(); ++it) {
-    if (iscntrl(*it))
+    if (iscntrl(*it) && !isspace(*it))
```

Comment:
hmm...I'll probably pull this change.  I did it because I was annoyed by the fact that I created a an attr with the data from /etc/fstab, but couldn't see the text because the newlines triggered a base64 encoding.